DQMOffline/Muon/interface/GEMOfflineDQMBase.h

```diff
@@ -97,7 +97,9 @@ class GEMOfflineDQMBase : public DQMEDAnalyzer {
 };
 
 inline int GEMOfflineDQMBase::getMaxVFAT(const int station) {
-  if (station == 1)
+  if (station == 0)
```

Comment:
Here you can make use of `GEMSubDetId`. `GEMSubDetId::tation(station) == GEMSubDetId::GE0`,

Reply:
Done